lib/chat_web/live/main_live/page/room.ex

```diff
@@ -151,6 +152,34 @@ defmodule ChatWeb.MainLive.Page.Room do
     end
   end
 
+  def load_new_messages(
+        %{
+          assigns: %{
+            room: room,
+            room_identity: identity,
+            last_loaded_index: index
+          }
+        } = socket
+      ) do
+    {_, new_messages} =
+      Rooms.read(room, identity, {nil, 0}, @per_page)
+      |> Enum.split_with(fn msg -> msg.index <= index end)
```

Comment:
It may be more than @per_page messages

we should read from `last_index` to newest. we might need a new method for this